src/main/java/edu/umich/ctools/sectionsUtilityTool/Friend.java

```diff
@@ -51,6 +53,7 @@
 	protected static String requesterEmailFile = null;
 	protected static String mailHost = null;
 	protected static String subjectLine = null;
+	protected static boolean mailDebug = false;
```

Comment:
Is this mailDebug or alsoPrintMailToLog?  There is a case for printing mail to log when it is not debug level if there is a problem in production.